src/function/matrix/eigs/complexEigs.js

```diff
@@ -514,19 +498,17 @@ export function createComplexEigs ({ addScalar, subtract, flatten, multiply, mul
     }
 
     // matrix is not diagonalizable
-    // compute off-diagonal elements of N = A - λI
-    // N₁₂ = 0 ⇒ S = ( N⃗₁, I⃗₁ )
-    // N₁₂ ≠ 0 ⇒ S = ( N⃗₂, I⃗₂ )
-
+    // compute diagonal elements of N = A - λI
     const na = subtract(a, l1)
-    const nb = subtract(b, l1)
-    const nc = subtract(c, l1)
     const nd = subtract(d, l1)
 
-    if (smaller(abs(nb), prec)) {
-      return [[na, one], [nc, zero]]
+    // N⃗₂ = 0 ⇒ S = ( N⃗₁, I⃗₁ )
```

Comment:
A thought: I think this file is the only one that contains non-ascii characters like `λ` and `N⃗₂ = 0 ⇒ S = ( N⃗₁, I⃗₁ )`. 

I haven't heard about issues with that in practice, but I can imagine that leads to weird cases. Maybe we should add a linter rule to check that we only use ascii characters to prevent possible issues in that regard. What do you think?

Reply:
OK, since these were bothering you, I converted the file to ascii (I think I got it all but didn't use any program to check).

But my two cents are: Unicode is explicitly allowed in JavaScript, and it is sometimes more compact/expressive, and the oldest PR that I want to get to back to as soon as "v13" is done will introduce the Unicode character √ into the read syntax for MEL (the mathjs expression language) , so at that point we'll be well and truly stuck with Unicode. So I don't see that it's worth anyone's effort to do anything to screen against it. If it passes the regression tests, I'm cool with unicode. I mean, some non-Roman non-Greek scripts might be difficult for _me_ to understand, but frankly I expect to live to a time when identifiers like 矢量 (according to the internet, means "vector" in Chinese) are commonplace in code packages, since I expect the world's fraction of Chinese-speaking/writing people will only increase...